source/MaterialXView/Viewer.h

```diff
@@ -362,6 +365,12 @@ class Viewer : public ng::Screen
     bool _bakeAverage;
     bool _bakeOptimize;
     bool _bakeRequested;
+
+    mx::FilePath _meshFilename;
```

Comment:
Unintended moving of this member variable within the header?

Reply:
Having order of member initialization problems in MSVC here.